.circleci/integration-tests/Dockerfile

```diff
@@ -112,10 +112,10 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
     && ./aws/install;
 
 # install AWS nuke
-RUN wget --quiet  \
-    "https://github.com/rebuy-de/aws-nuke/releases/download/${AWS_NUKE_VERSION}/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz" \
-    && tar -xzvf aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz -C /usr/local/bin \
-    && mv /usr/local/bin/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64 /usr/local/bin/aws-nuke
+#RUN wget --quiet  \
+#    "https://github.com/rebuy-de/aws-nuke/releases/download/${AWS_NUKE_VERSION}/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz" \
+#    && tar -xzvf aws-nuke-${AWS_NUKE_VERSION}-linux-amd64.tar.gz -C /usr/local/bin \
+#    && mv /usr/local/bin/aws-nuke-${AWS_NUKE_VERSION}-linux-amd64 /usr/local/bin/aws-nuke
```

Comment:
Any reason for commenting this out?

Reply:
I forgot to submit my review . The reason is below:
Commenting this out as the command is failing for images built from runtime images as base image. Need to investigate how to fix this.


<img width="1060" alt="Screenshot 2022-05-06 at 8 29 21 PM" src="https://user-images.githubusercontent.com/10206082/167159219-806777a1-d74f-4813-9ac2-4e285066c490.png">

Reply:
Commenting this out as the command is failing for images built from runtime images as base image. Need to investigate how to fix this.

Reply:
i am checking this issue with aws-nuke

Reply:
Since https://github.com/astronomer/astronomer-providers/pull/323 is merged, we can uncomment it?

Reply:
yes, perfect. Thank you @phanikumv & @kaxil